alembic/versions/33de75e55858_adding_datetime_column.py

```diff
@@ -0,0 +1,44 @@
+"""Adding datetime column
+
+Revision ID: 33de75e55858
+Revises: 525b125ec799
+Create Date: 2015-10-10 12:32:39.936107
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = '33de75e55858'
+down_revision = '525b125ec799'
+
+from alembic import op
+import sqlalchemy as sa
+
+
+def upgrade():
+    ### commands auto generated by Alembic - please adjust! ###
+    op.add_column('campaign_anon_view', sa.Column('datetime', sa.DateTime(), nullable=False))
```

Comment:
When you add a non-nullable column to a table with existing data, you have to provide a default value or the operation will fail. The correct way to add this column is:

1. Add a nullable column
2. Update the column in all rows to have the same value as the `created_at` column
3. Edit the column to make it non-nullable